rust/src/types.rs

```diff
@@ -3166,6 +3166,11 @@ impl ParamVar {
     }
   }
 
+  pub fn set_cloning(&mut self, value: &Var) {
```

Comment:
Rename this to set, and the other one to some set_unsafe, set_fast, set_direct or something. Mostly always it should be cloned

Reply:
Ideally we can have the fast path implemented in a reusable way in rust as well, and fall back to the cloneVar FFI, but for now renaming is fine.